common/k8stest/util_deploy.go

```diff
@@ -466,3 +466,14 @@ func RestoreDeploymentReplicas(deploymentName string, namespace string, timeout_
 	}
 	return err
 }
+
+func CheckDeploymentRunning(name, namespace string, expectedReplicas int) (bool, error) {
```

Comment:
Rename `CheckDeploymentRunning` to something related to expected replica check like `VerifyDeploymentReadyReplicaCount`

Reply:
Fixed